modules/setting/setting.go

```diff
@@ -899,6 +901,10 @@ func NewContext() {
 		log.Fatal("Failed to map Metrics settings: %v", err)
 	}
 
+	u := *appURL
```

Comment:
De-referenced the pointer in case anyone uses `appURL` in a later change, since right below this I fudge the `u.Path` manually.